libraries/cli/include/cli/default_config.hpp

```diff
@@ -16,6 +16,8 @@ const char *default_json = R"foo({
   "network_sync_level_size": 25,
```

Comment:
we have now also mainnet_config.hpp, which should be propably adjusted too

Reply:
Fixed